docs/decisions/008-themes.md

```diff
@@ -0,0 +1,55 @@
+# Themes
+
+Date: 2024-10-08
+
+Status: accepted
+
+## Context
+
+Since the very beginning, the Spirit Design System has been using the concept of inverted tokens and component variants
+to enable light content on dark backgrounds.
+
+However, this approach has several drawbacks:
+
+1. **Limited component support**: The inverted component variant need to be created manually for each component. This
+   leads to a limited set of components that support the inverted theme.
+2. **Number of tokens**: The inverted tokens are hard to maintain and can lead to inconsistencies. For example, the text
+   color token `text-primary-default` needs an inverted variant `text-primary-inverted-default`. This leads to a lot of
+   duplicated tokens and makes it hard to keep them in sync.
+3. **Manual composition**: In a larger inverted composition, all components need to be switched to the inverted variant
+   manually.
+4. **Dark mode (un)readiness**: The concept of the inverted tokens and component variants is not flexible enough to
+   support a dark mode. The name "inverted" is too specific and does not work for a dark mode where, from the design
+   point of view, the inversion cannot be really applied.
+
+## Decision
+
+1. **Removal of inverted tokens and component variants.** In some cases, the inverted tokens will be replaced by new
+   "neutral" tokens that are more flexible and can be used in both light and dark themes. In other cases, themes take
+   over the responsibility of defining the colors.
+2. **Introduction of themes.** To enable component independent, **scoped changes of appearance**, themes will be
+   introduced. Themes are another layer of abstraction that can be applied to any design token. We decided to start with
+   theming the colors and introduce a light and an on-brand theme for branded sections like site header and cover
```

Comment:
I think it would be also nice to know what brand colors and on-brand theme mean. It looks like it is used without any context.

Reply:
I am intentionally not being specific here. I think about the decision as if it had been written **before** it was implemented. We didn't have the details at that moment. Or maybe I got you wrong…?

Reply:
I mean that we are talking about brand and on-brand colors. We know what it means. But the developer or designer outside our theme will know nothing about it. So I suggest explaining briefly what brand and on-brand themes/colors are. Maybe a note in a footer or just in brackets.

Reply:
OK, and now? I don't think I could be more specific… 🙂